opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/ServerTracingAutoConfigurationDefaultsTest.java

```diff
@@ -115,6 +122,17 @@ public void testExcluded() throws InterruptedException {
         assertThat(Mockito.mockingDetails(mockDecorator2).getInvocations()).hasSize(0);
     }
 
+    // Test that /actuator/info is excluded due to the default skipPattern
+    @Test
+    public void testActuatorExcluded() throws InterruptedException {
```

Comment:
Can you please also change the name of `testExcluded` to `testInfoExcluded`?

Reply:
Yes of course, It's done, thank you for the feedback.

Reply:
Thanks! I'll merge this once CI goes green.